docs/stat.py

```diff
@@ -48,12 +90,19 @@
 modelzoo = f"""
 # Model Zoo Statistics
 
-* Number of checkpoints: {len(allckpts)}
-* Number of configs: {len(allconfigs)}
-* Number of papers: {len(allpapers)}
+Number of checkpoints: {len(allckpts)}</br>
```

Comment:
why change these

Reply:
A models list is to be placed below these three lines of statistics and each model has a paperlist

Sphinx has some problems in rendering tables and unordered lists. Specifically,
If we set the model list to level 2 unordered list or higher, then after inserting a table, the remaining level 2 unordered list will be rendered as `code style`.
If we indent the table code, the table will be rendered as `code style`.

So I think I must set the model list as level 1 unordered list and thus I didn't set the three statistics as unordered list.